packages/p2p/src/stores/order-store.js

```diff
@@ -271,7 +271,7 @@ export default class OrderStore {
                     }
                 } else if (response?.error?.code === api_error_codes.PERMISSION_DENIED) {
                     this.root_store.general_store.setIsBlocked(true);
-                } else {
+                } else if (response.error.code !== api_error_codes.ADVERTISER_NOT_FOUND) {
```

Comment:
`} else if (response?.error.code !==`

Reply:
we are already checking for `response` above